include/kframework/gas.md

```diff
@@ -0,0 +1,55 @@
+KEVM Gas
+========
+
+Here is a representation of `Gas`. This sort is used by `<gas>`, `<callGas>`, and `<gasUsed>` cells of the EVM configuration. As is in this file, `Gas` is a super-sort of `Int` with no differing behaviour. However the distinction at this level allows for an extension of infinite gas later, see `infinite-gas.md`.
+
+```k
+module GAS-SYNTAX
+    imports INT-SYNTAX
+    
+    syntax Gas ::= Int
+    
+    syntax Gas ::= Gas "*Gas" Gas [function, left, total]
+                 | Gas "/Gas" Gas [function, left]
+                 > left:
+                   Gas "+Gas" Gas [function, left, total]
+                 | Gas "-Gas" Gas [function, left, total]
+
+    syntax Bool ::= Gas  "<Gas" Gas [function, left, total]
+                  | Gas "<=Gas" Gas [function, left, total]
+                  | Gas  ">Gas" Gas [function, left, total]
+                  | Gas ">=Gas" Gas [function, left, total]
+    
+    syntax Gas ::= "minGas"  "(" Gas "," Gas ")" [function, total]
+    syntax Int ::= "gas2Int" "(" Gas ")"         [function, total]
```

Comment:
Group the `gas2Int` definition with the definition of the subsort of `Int` into `Gas` at the top of the module, and group the `minGas` declaration with the other gas-returning operators above.